django/test/testcases.py

```diff
@@ -274,6 +274,12 @@ def __call__(self, result=None):
         """
         self._setup_and_call(result)
 
+    def __getstate__(self):
+        """
+        Make SimpleTestCase pickable for parallel tests using subtests.
+        """
+        return {k: v for k, v in self.__dict__.items() if k.startswith("_")}
```

Comment:
Does it affect attributes without the leading underscore? e.g. `failureException`, `longMessage`, or `maxDiff`.

Reply:
Indeed, thanks! 👍  I changed the condition to select the attributes we want to pickle